src/Microsoft.Health.Dicom.Core/Features/Retrieve/RetrieveResourceService.cs

```diff
@@ -89,7 +89,7 @@ public async Task<RetrieveResourceResponse> GetInstanceResourceAsync(RetrieveRes
                 message.AcceptHeaders);
 
             string requestedTransferSyntax = validAcceptHeader.TransferSyntax.Value;
-            bool isOriginalTransferSyntaxRequested = DicomTransferSyntaxUids.IsOriginalTransferSyntaxRequested(requestedTransferSyntax);
+            bool isOriginalTransferSyntaxRequested = DicomTransferSyntaxUids.IsAnyTransferSyntaxRequested(requestedTransferSyntax);
```

Comment:
I think Any is confusing. Original is more accurate. Any means we can transcode and do anything. Original means we are returning what was stored.

here is a example of another repo where same naming convesion is used,

Reply:
There's two concepts here that I think should be distinguished, customer input and our choice: 
- The customer, by sending `*`, is requesting any transfer syntax according to the standard.
- Our service is choosing to break ties by returning the original transfer syntax, but this is not strictly what the customer is requesting - we are making that decision.  

The goal here is to clarify what is the customer input and what is our choice, because someone looking at the code to understand how we pick return syntax will need to understand that distinction.